numpy/array_api/linalg.py

```diff
@@ -318,9 +318,9 @@ def _solve(a, b):
     # This does nothing currently but is left in because it will be relevant
     # when complex dtype support is added to the spec in 2022.
     signature = 'DD->D' if isComplexType(t) else 'dd->d'
-    with errstate(call=_raise_linalgerror_singular, invalid='call',
-                  over='ignore', divide='ignore', under='ignore'):
-        r = gufunc(a, b, signature=signature, extobj=extobj)
+    with np.errstate(call=_raise_linalgerror_singular, invalid='call',
```

Comment:
Interesting that the errstate removal did make it?  Anyay, LGTM, thanks.

Reply:
I think the implementation here was added before this was thoroughly tested in the test suite. Maybe the `errstate` import was removed and that went unnoticed due to incomplete test coverage.